examples/data-router/src/app.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {
+  createBrowserRouter,
+  createRoutesFromChildren,
+  Route,
+  RouterProvider,
+} from "react-router-dom";
+
+import {
+  Fallback,
+  Layout,
+  homeLoader,
+  Home,
+  deferredLoader,
+  DeferredPage,
+  deferredChildLoader,
+  deferredChildAction,
+  DeferredChild,
+  todosAction,
+  todosLoader,
+  TodosList,
+  TodosBoundary,
+  todoLoader,
+  Todo,
+  sleep,
+  AwaitPage,
+} from "./routes";
+import "./index.css";
+
+let router = createBrowserRouter(
+  createRoutesFromChildren(
```

Comment:
Let's alias this to `createRoutesFromElements` and one day deprecate the other name. `children` is the wrong name in this context now.

Reply:
https://github.com/remix-run/react-router/pull/9227/commits/433db8bd33bc655d2148047e1ea99a2ba33aa828